### PR TITLE
Packaging + CI

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: mediawiki_matrix_bot/__init__.py [hunk 1]
```diff
@@ -124,17 +125,12 @@ async def forward_news(client, room, message_obj,baseurl):
         )
 
 
-async def fetch_changes(baseurl):
+async def fetch_changes(baseurl: str) -> Any:
     async with aiohttp.ClientSession() as session:
         async with session.get(f'{baseurl}/api.php?action=query&list=recentchanges&format=json&rcprop=user|comment|flags|title|sizes|loginfo|ids|revision') as response:
-            return await response.json()
```
Comment (makefu) [inline mediawiki_matrix_bot/__init__.py:130]: "official" documentation contained the await, not sure if it is actually required

File: mediawiki_matrix_bot/__init__.py [hunk 2]
```diff
@@ -107,7 +108,7 @@ def format_data(obj,baseurl,udpinput=False,):
         f" {bold(diff_length)} {color(comment,'cyan')}"
 
 
-async def forward_news(client, room, message_obj,baseurl):
+async def forward_news(client: AsyncClient, room: str, message_obj: Dict[str, Any],baseurl: str) -> None:
```
Comment (makefu) [inline mediawiki_matrix_bot/__init__.py:111]: so clean, thanks! :+1:

**Review by makefu — APPROVED** [review]

File: mediawiki_matrix_bot/__init__.py (hunk 1 above)
Comment (makefu) [reply mediawiki_matrix_bot/__init__.py:130]: https://docs.aiohttp.org/en/stable/client_quickstart.html

Comment (mweinelt) [reply mediawiki_matrix_bot/__init__.py:130]: Making the function async and then Not awaiting a blocking call makes little sense.

Comment (Mic92) [reply mediawiki_matrix_bot/__init__.py:130]: Ah, sorry. I was trying to fix the type but than forgot to re-add it again.

**Comment by Mic92** [comment]
Done.

**Pull request merged**